yeat/Snakefile

```diff
@@ -79,15 +80,27 @@ rule downsample:
         read1="seq/fastp/{sample}.R1.fq.gz",
         read2="seq/fastp/{sample}.R2.fq.gz",
         mash_report="seq/mash/{sample}.report.tsv"
+    params:
+        downsample=config["downsample"]
     run:
+        if params.downsample == -1:
+            p = Path("seq/downsample")
+            p.mkdir(parents=True, exist_ok=True)
+            copyfile(input.read1, output.sub1)
+            copyfile(input.read2, output.sub2)
+            return
```

Comment:
Learned that you actually can `symlink` instead of copy files; however, the current snakemake version is unable to do that until 3.8. (We are currently on 3.7.)

```
Path(output.sub1).symlink_to(Path(input.read1))
Path(output.sub2).symlink_to(Path(input.read2)) 
```

![image](https://user-images.githubusercontent.com/33472323/192024628-5862d8d2-207f-4e99-9490-425c5d65f9c7.png)

Reply:
Don't confuse Python 3.7 with Snakemake 3.7. We support Python 3.7 for many of our workflows, but Snakemake 3.7 is *ancient* in bioinformatics years. :smiley: At the moment I have `snakemake>=6.0,<7.0` pinned on several repos, since there seem to be some error handling regressions with Snakemake version 7.